cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -87,6 +87,7 @@ angular.module(PKG.name + '.commons')
           params = {};
           angular.copy($scope.params, params);
           params.max = 50;
+          params.escape = false;
```

Comment:
this is not redundant, compared to your change on line 70?
Why are both necessary?